app/controllers/CampaignBotController.js

```diff
@@ -179,6 +180,13 @@ class CampaignBotController {
   postReportback(req) {
     this.debug(req, 'postReportback');
 
+    rabbit.produce('activity events', {
+      type: 'reportback',
+      userId: req.user._id,
+      campaignId: req.campaign._id,
+      campaignRunId: req.campaign.current_run,
+    });
```

Comment:
@aaronschachter I did not see that issue, thanks for pointing that out! I'll move this over to Signup then